manga_translator/translators/chatgpt.py

```diff
@@ -160,3 +173,8 @@ async def _request_translation(self, prompt: str) -> str:
 
         # If no response with text is found, return the first response's content (which may be empty)
         return response.choices[0].message.content
+
+class GPT35TurboTranslator(GPTChatTranslator):
+    _model = 'gpt-3.5-turbo'
+class GPT4Translator(GPTChatTranslator):
+    _model = 'gpt-4'
```

Comment:
Can you revert the creation of GPTChatTranslator? I dont see any point in abstracting this away as i would like to keep the ability to make easy deviations in the usage of openai.ChatCompletion.create.